src/app/routes/connector-ui/policy-definition-page/new-policy-dialog/new-policy-dialog.component.ts

```diff
@@ -25,13 +26,14 @@ export class NewPolicyDialogComponent implements OnDestroy {
     private dialogRef: MatDialogRef<NewPolicyDialogComponent>,
     public validationMessages: ValidationMessages,
     private policyDefinitionBuilder: PolicyDefinitionBuilder,
-  ) {}
+  ) {
+    this.minDate = new Date();
+  }
```

Comment:
Hmm, nice idea, but we really should not enforce that. If somebody adds a data offer and wants to demonstratively say, that it's available starting the start of a real-life contract or something, then he should be able to set the date to monday, even if it's tuesday.